synapse/tests/test_lib_grammar.py

```diff
@@ -910,8 +925,8 @@ def test_grammar(self):
         for i, query in enumerate(_Queries):
             try:
                 tree = parser.parse(query)
-                # print(f'#{i}: {query}')
-                # print(tree.pretty(), '\n')
+                print(f'#{i}: {query}')
```

Comment:
change in skipped test?

Reply:
It is now a slightly more useful skipped test